examples/webgpu_mrt_readback.html

```diff
@@ -0,0 +1,259 @@
+
+				renderTarget = new THREE.RenderTarget(
+					window.innerWidth * window.devicePixelRatio,
+					window.innerHeight * window.devicePixelRatio,
+					{ count: 2, minFilter: THREE.NearestFilter, magFilter: THREE.NearestFilter }
+				);
+
+				// Name our G-Buffer attachments for debugging
+
+				renderTarget.textures[ 0 ].name = 'diffuse';
+				renderTarget.textures[ 1 ].name = 'normal';
+
+
+				// Init readback render target, readback data texture, readback material
+				// Be careful with the size! 512 is already big. Reading data back from the GPU is computationally intensive
+
+				size = 512;	
+				readbackTarget = new THREE.RenderTarget(size, size, { count: 2 });
+
+				pixelBuffer = new Uint8Array(size ** 2 * 4).fill(0);
+				pixelBufferTexture = new THREE.DataTexture(pixelBuffer, size, size); 
+				pixelBufferTexture.type = THREE.UnsignedByteType,      
+				pixelBufferTexture.format = THREE.RGBAFormat,
+
+				readbackMaterial = new MeshBasicNodeMaterial();
+
+
+				// Scene setup
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0x222222 );
+
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 0.1, 50 );
+				camera.position.z = 4;
+
+
+				const loader = new THREE.TextureLoader();
+
+				const diffuse = loader.load( 'textures/hardwood2_diffuse.jpg' );
+				diffuse.colorSpace = THREE.SRGBColorSpace;
+				diffuse.wrapS = THREE.RepeatWrapping;
+				diffuse.wrapT = THREE.RepeatWrapping;
+
+				torus = new THREE.Mesh(
+					new THREE.TorusKnotGeometry( 1, 0.3, 128, 32 ),
+					new WriteGBufferMaterial( diffuse )
+				);
+
+				scene.add( torus );
+
+
+				material = new ReadGBufferMaterial( renderTarget.textures[ 0 ], renderTarget.textures[ 1 ] );
+				quadMesh = new QuadMesh( material );
+
+				// Controls
+
+				new OrbitControls( camera, renderer.domElement );
+
+				window.addEventListener( 'resize', onWindowResize );
+
+			}
+
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+				const dpr = renderer.getPixelRatio();
+				renderTarget.setSize( window.innerWidth * dpr, window.innerHeight * dpr );
+
+			}
+
+			async function render( time ) {
+
+				torus.rotation.y = ( time / 1000 ) * .4;
+
+				// render scene into target
+				renderer.setRenderTarget( renderTarget );
+				renderer.render( scene, camera );
+
+				// render post FX
+				renderer.setRenderTarget( null );
+
+				if(selector == 1){
+					quadMesh.material = material;
+					quadMesh.render( renderer );
+				}
+
+				await readback();
+			}
+
+
+			async function readback(){
+
+				renderer.setRenderTarget( readbackTarget );
+				renderer.render( scene, camera );
+				renderer.setRenderTarget( null );
+
+				const width = readbackTarget.width;
+				const height = readbackTarget.height;
+
+				if(selector == 2){
```

Comment:
The formatting of the code is still not correct. Please make sure ESLint properly lints the code according to the project's code style.

Reply:
My apologies, I thought I did that with the rebase. This is still new for me and I admit there is still a lot I don't know about github.
By formatting do you mean 
if( selector == 2 ) { 
instead of 
if(selector == 2){ 
?

Reply:
Yes. And ideally use `===` for strict comparison.

Don't worry if you need time to get things going.

Reply:
Thanks for your understanding. I can easily solve the formatting issue. About the rebase, is there anyone I can ask how I do it correctly?

Reply:
You can use the merge or rebase command. Normally you start by fetching the latest commit from the upstream repository:
```
git fetch upstream
```
You can merge the latest changes on `dev` into your branch:
```
git merge upstream/dev
```
Or perform a rebase:
```
git rebase upstream/dev
```
The are different resources online which explain the difference between merge and rebase. Both have their pros and cons. However, rebase is ideal for situations when you want to move the point where you have branched to a new starting point. This is exactly what you are looking for in this instance.